docs/components/DateTimePicker.tsx

```diff
@@ -0,0 +1,319 @@
+  // We do this because the ReactDatePicker localizes the date to our current computer. But we don't
+  // want that localization if we are looking at a time in another timezone.
+  const dateWithOffset: Date = useMemo(() => {
+    return DateTime.fromISO(date)
+      .minus({ minutes: offsetFromCurrentTz })
+      .toJSDate();
+  }, [offsetFromCurrentTz, date]);
+
+  return (
+    <div className="time-pickers">
+      <ReactDatePicker
+        disabled={false}
+        minDate={minDate ? new Date(minDate) : null}
+        selected={dateWithOffset}
+        selectsRange={false}
+        onChange={(date) => {
+          if (date instanceof Date) {
+            // We adjust for the localization mentioned above.
+            const startDate = DateTime.fromJSDate(date).plus({
+              minutes: offsetFromCurrentTz,
+            });
+
+            setDate(startDate.toISO());
+          }
+        }}
+        className="luma-input"
+        wrapperClassName="date-picker-wrapper mr-2"
+        dateFormat="MMMM d, yyyy"
+      />
+      <ReactDatePicker
+        disabled={false}
+        selected={dateWithOffset}
+        onBlur={(e) => {
+          const duration = parseTimeFromString(e.target.value);
+          if (duration) {
+            setDate(
+              DateTime.fromISO(date)
+                .startOf("day")
+                .plus(duration)
+                .plus({ minutes: offsetFromCurrentTz })
+                .toISO()
+            );
+          }
+        }}
+        onChange={(newDate) => {
+          // There is a bug that causes the date to get reset sometimes
+          // So we manually preserve the date and just add on the time
+          if (newDate && newDate instanceof Date) {
+            // The new date we get is in our current timezone so when we get the minute offset,
+            // we do it in the current timezone.
+            const startAtDt = DateTime.fromJSDate(newDate, {
+              zone: myTimezone!,
+            }).startOf("day");
+
+            const minutesIntoDay = Math.floor(
+              DateTime.fromJSDate(newDate, {
+                zone: myTimezone!,
+              }).diff(startAtDt, "minutes").minutes
+            );
+
+            const currentTimezone = timezone || myTimezone!;
+            const currentDt = DateTime.fromISO(date, { zone: currentTimezone });
+            const newDt = DateTime.fromObject(
+              {
+                year: currentDt.year,
+                month: currentDt.month,
+                day: currentDt.day,
+                hour: Math.floor(minutesIntoDay / 60),
+                minute: minutesIntoDay % 60,
+              },
+              { zone: currentTimezone }
+            );
+
+            setDate(newDt.toISO());
+          }
+        }}
+        showTimeSelect
+        showTimeSelectOnly
+        className="luma-input time-input"
+        wrapperClassName="time-picker-wrapper"
+        dateFormat="h:mm aa"
+      />
+
+      <style jsx global>{`
```

Comment:
Ahh crap, I don't think this is going to work since it's SCSS and we are trying to only use CSS in Styled JSX in NFToken Docs. I can try to fix this in the main Luma repo.

Reply:
So shall I also fix it here in the meantime? I mean, do I understand correctly that we want to rewrite it to pure CSS while still keeping using Styled JSX?

Reply:
I have updated in Luma, can you update `@luma-team/lux-sass` version and pull over the new components from `lux-atoms`